src/doc/book/concurrency.md

```diff
@@ -281,8 +281,8 @@ And... still gives us an error.
 ```
 
 `Arc<T>` by default has immutable contents. It allows the _sharing_ of data
-between threads, but shared mutable data is unsafe and when threads are
-involved can cause data races!
+between threads, but shared mutable data is unsafe-and when threads are
+involved-can cause data races!
```

Comment:
This is a *very* minor detail, but is it possible you could use the 'dash' character (—) instead of a 'hyphen' character (-) for this?

https://owl.english.purdue.edu/engagement/3/7/98/